### PR TITLE
We accidently removed RFC3987 validation from route paths

### PR DESCRIPTION
**Pull request by tyler-ball**
### Summary
Adding to known issues

### Testing
Namely

### CODE REVIEW

File: app/gateway/changelog.md [hunk 1]
```diff
@@ -769,6 +769,8 @@ openid-connect
   recommended to upgrade yet if you are an OpenTracing user. This will be
   rectified in an upcoming patch/minor release.
 
+* RFC3987 validation on route paths was removed, allowing operators to create a route with an invalid path URI like `/something|`. This validation will be added back in a future release.
```
Comment (tyler-ball) [inline app/gateway/changelog.md:772]: > operators

What is the correct way to refer to someone who configures Kong (I originally wrote `kong user` but that doesn't quite seem right).

Comment (lena-larionova) [reply app/gateway/changelog.md:772]: We usually say simply "users". Operators works too though.

**Review by lena-larionova — APPROVED** [review]
LGTM

File: app/gateway/changelog.md (hunk 1 above)
Comment (dndx) [inline app/gateway/changelog.md:772]:
```suggestion
* RFC3987 validation on route paths was removed, allowing operators to create a route with an invalid path
  URI like `/something|` which can not match any incoming request. This validation will be added back in a future release.
```

Some line width control is recommended.

**Pull request merged**